Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.10-slim-bullseye
+
+WORKDIR /app
+COPY requirements.txt requirements.txt
+RUN python3 -m pip install --upgrade pip && pip install -r requirements.txt
+COPY . .
```

Comment:
Can this copy command on line 6 be moved in place of line 4 ?

Reply:
I think separating them might improve caching but does seem to still work file after this change so I have implemented it.

Reply:
@Adam-W1 OK this explains why its done like this to improve caching of layers https://stackoverflow.com/a/34399661/238230 so I think we should revert to the previous approach.